scripts/core/loader.yml

```diff
@@ -187,6 +186,31 @@ params:
       name: pUserData
       desc: "[in][out][optional] pointer to data to be passed to callback."
 --- #--------------------------------------------------------------------------
+type: fptr_typedef
```

Comment:
This is unused?

Reply:
I'm still using the callback function pointer definition in the [callback helpers](https://github.com/oneapi-src/unified-runtime/pull/1780/files#diff-9355e740a21703116ed14e0e168aabc2c999b47a27f6dd934a65b658a95a4981R67)

Reply:
In which case you should be able to simply define that type there. Or do you want that type to be available without the callback helper header?